src/main/scala/com/codacy/client/bitbucket/v2/Repository.scala

```diff
@@ -8,7 +8,11 @@ import play.api.libs.json._
 case class Repository(name: String, full_name: String, description: String, scm: String,
                       created_on: LocalDateTime, updated_on: LocalDateTime, owner: String, size: Long,
                       has_issues: Boolean, is_private: Boolean, language: String,
-                      url: Seq[RepositoryUrl])
+                      url: Seq[RepositoryUrl]) {
```

Comment:
maybe we should use the plural form here, since it is a list of urls.

Reply:
Done